RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,59 @@
+# Releasing bittensor
+
+The release manager in charge can release a Bittensor version using the script:
+  - [./scripts/release/release.sh](./scripts/release/release.sh)
+
+## Release script usage
+
+Options:
+  - -V, --version: type of update. It could be major, minor or patch.
+  - -A, --apply: This specify to apply the release. Without this the release will just show a dry run with no changes.
+  - -T,--github-token: A github personal access token to interact with the Github API.
+
+### Github token
+
+Since you need to use a secret when releasing bittensor (github personal access token), I encourage you to use [pass](https://www.passwordstore.org/) or a similar tool that allows you to store the secret safely and not expose it in the history of the machine you use.
+
+So you can have:
+```
+GITHUB_ACCESS_TOKEN=$(pass github/your_personal_token_with_permisions)
+```
+
+or
+```
+GITHUB_ACCESS_TOKEN=$(whatever you need to get the token safely)
+```
+
+### Executions
+
+So, executing the script to release a minor version will be:
+
+```
+# For a dry run
+./scripts/release/release.sh --version minor --github-token $GITHUB_ACCESS_TOKEN`
+```
+
+```
+# Applying changes
+./scripts/release/release.sh --apply --version minor --github-token $GITHUB_ACCESS_TOKEN`
+```
+
+## Checking release
+
+After the execution of the release script we would have generated:
+  - A new git tag in [github.com](https://github.com/opentensor/bittensor/tags)
+  - A new github release in [github.com](https://github.com/opentensor/bittensor/releases)
+  - A new pip package in [pypi.org](https://pypi.org/project/bittensor/#history)
+  - A new docker image in [hub.docker.com](https://hub.docker.com/r/opentensorfdn/bittensor/tags)
+
+## After release
+
+After a Bittensor release we have to
+- Update [cubit](https://github.com/opentensor/cubit).
```

Comment:
How about cubit? Does that need to be updated after each release?

Reply:
Yes, right now it's a separated task we have to do around the cubit repository as I did for the 3.4.3 version [in this PR](https://github.com/opentensor/cubit/pull/3/files)

Reply:
Eventually we are going to have this wiring automated